tests/playwright/sanity/container.test.js

```diff
@@ -261,4 +261,56 @@ test.describe( 'Container tests', () => {
 			container: false,
 		} );
 	} );
+
+	test( 'Fallback image is not on top of background video', async ( { page }, testInfo ) => {
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		await wpAdmin.setExperiments( {
+			container: true,
+		} );
+
+		await page.goto( '/wp-admin/media-new.php' );
+
+		if ( await page.locator( 'text=browser uploader' ).nth( 1 ).isVisible() ) {
+			await page.locator( 'text=browser uploader' ).nth( 1 ).click();
+		}
+
+		await page.setInputFiles( 'input[name="async-upload"]', './tests/playwright/resources/video.webm' );
+		await page.locator( '#html-upload' ).click();
+		await page.waitForSelector( '.attachments-wrapper' );
+		await page.locator( 'ul.attachments li' ).nth( 0 ).click();
```

Comment:
nth is not the best option

Reply:
@TzviRabinovitch nth cannot be avoided here.